pyspectral/near_infrared_reflectance.py

```diff
@@ -263,10 +265,13 @@ def reflectance_from_tbs(self, sun_zenith, tb_near_ir, tb_thermal, **kwargs):
         if l_nir.ravel().shape[0] < 10:
             LOG.info('l_nir = %s', str(l_nir))
 
-        sunzmask = (sun_zenith < 0.0) | (sun_zenith > 88.0)
-        sunz = sun_zenith.clip(0, 88.0)
-
+        LOG.debug("Apply sun-zenith angle clipping between 0 and %5.2f", self.sunz_threshold)
+        sunzmask = (sun_zenith < 0.0) | (sun_zenith > self.sunz_threshold)
```

Comment:
What does SZA < 0 mean? Could that happen with some data that the SZA gets illogical values?

Reply:
Good question @pnuu , don't remember why it is there, but I assume we put it there for a reason. I could add a debug printout saying if any value was below zero perhaps?